internal/controllers/provisioningrequest_controller.go

```diff
@@ -79,6 +80,15 @@ type nodeInfo struct {
 	interfaces     []*hwv1alpha1.Interface
 }
 
+// timeouts holds the timeout values, in minutes,
+// for hardware provisioning, cluster provisioning
+// and cluster configuration.
+type timeouts struct {
+	hardwareProvisioningMinutes int
+	clusterProvisioningMinutes  int
+	configurationMinutes        int
```

Comment:
Should these be time.Duration instead? And remove "Minutes" from the names.

Reply:
Yeah, I feel it's clearer to use time.Duration to represent the timeout. I had them as time.Duration before, so I will change them back.